viewer/packages/api/src/public/migration/Cognite3DViewer.ts

```diff
@@ -243,8 +243,7 @@ export class Cognite3DViewer {
         this._renderer,
         this._sceneHandler,
         revealOptions,
-        options.customDataSource,
-        options.sdk
+        options.customDataSource
```

Comment:
What's going on here? I can see a world where users implement a local cache for models, but still query and use the SDK against CDF for certain scenarios (such as node styling sets). But perhaps this isn't supported now either?

Reply:
I agree that this is a valid use case, but the current way of supporting such operations, is to provide a `DataSource` object to Reveal, which already contains a model data provider and a node styling provider. In any case, passing the SDK here is unnecessary.